docs/bis/ec/ec-format.md

```diff
@@ -101,7 +101,12 @@ Supported scientific type:
 
 **includeZero** determines whether to keep a segment of the composite even if its magnitude is zero. By default, this is true.
 
-**units (1..4)** an optional array of unit + label override objects.  A single units entry formats a value like `42.42 ft`.  If multiple units are specified they should be in decreasing magnitude and value split among the units like `42 ft 5.02 in`
+**units (1..4)** an optional array of unit + label override objects.
+  - A single units entry formats a value like `42.42 ft`.
+  - If multiple units are specified they should be in decreasing magnitude and value split among the units like `42 ft 5.02 in`.
+  - Unit labels are optional. If not specified, the label will be set as the unit's diplay label.
+  - Unit label can be specified as an empty string.
```

Comment:
Document what happens when an empty string is applied, e.g. that no label will be shown for that unit